java/arcs/sdk/examples/unit_testing/ComputePeopleStats.kt

```diff
@@ -0,0 +1,27 @@
+package arcs.sdk.examples.unit_testing
```

Comment:
nit, I think it's advised to not use underscores in java/kotlin package names

Reply:
Done!